### PR TITLE
added support for primitive wrappers to not be handled as objects

### PR DESCRIPTION
**Pull request by jej2003**

### CODE REVIEW

**Comment by jej2003** [comment]
closes #48

**Comment by jej2003** [comment]
@briancavalier , just checking in to see if there's anything that I can do to help move this forward?  Perhaps there is a better way to implement?

**Comment by briancavalier** [comment]
Hey @jej2003 , thanks for the PR! I should have time today to take a look.

File: lib/clone.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ function clone(x) {
 		return x;
 	}
 
+	if (typeof x === 'object' && x !== null && (x instanceof String || x instanceof Number || x instanceof Boolean)) {
```
Comment (briancavalier) [inline lib/clone.js:17]: I think we can simplify slightly: `instanceof` is sufficient, and the previous `if` rules out `null` and non-objects.

```suggestion
	if (x instanceof String || x instanceof Number || x instanceof Boolean) {
```

**Review by briancavalier — CHANGES_REQUESTED** [review]
Looks good! Just one minor suggestion.

**Comment by jej2003** [comment]
@briancavalier agree and applied the requested change. Thanks!

**Review by briancavalier — APPROVED** [review]
Thanks @jej2003!

**Comment by briancavalier** [comment]
I'll try to publish a new version in the next couple days.

**Comment by briancavalier** [comment]
@jej2003 Apologies for not publishing a new version yet.

NPM support is helping us with an issue that's preventing us from publishing. I'll let you know when we've published a new version.

**Pull request merged**